src/converter.js

```diff
@@ -178,32 +179,38 @@ converter._object = (name, node, mode) => {
         node
       )
     }
-    if (!_.isPlainObject(node.properties)) {
-      throw new SchemaError('No properties defined for object', node)
+    if (!hasProperties) {
+      // Big Query can handle semi-structured data :
+      // https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-json#loading_semi-structured_json_data
+      fieldType = 'JSON'
     }
-    if (Object.keys(node.properties).length === 0) {
+
+    if (hasProperties && Object.keys(node.properties).length === 0) {
```

Comment:
Could we use an else if here to avoid having to check `hasProperties` again?

Reply:
absolutely !